pages/docs/concepts/message.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Message
+weight: 10
+---
+
+## What is a message?
+A `message` is a communication asset used to transmit or exchange information from a sender to the receiver through `channels`. A single `message` can be consumed by multiple independent receivers and can also be defined as an _event_ or _command_. The sender includes a payload of data that needs to be processed and serialized by the receiver into an appropriate format, e.g., JSON, XML, binary, etc. It may also include metadata, i.e., information that describes the message itself. This metadata is often known as _headers_ or _properties_.
+
+
+``` mermaid
+graph LR
+subgraph one[channel]
+b([message])
+end
+
+A{{sender application}} --> b --> C{{receiver application}}
+  
+```
+
+In the diagram above, the sender application transmits a `message` to the receiver application.
+
+## Messages vs Events
+A `message` carries information from one application to the other, while an `event` is a message that provides details of something that has already occurred. `Messages` contain all necessary information required to perform the request, while `events` only contain data of the changed state.
+Overall, `events` are `messages` but not all `messages` are `events`.
+
```

Comment:
Hey @thulieblack, on the completion of this page or the pages you add, please make sure that you add the `Prev` and `UpNext` buttons similar to other pages and update the routes in such a way that it links with the Docs Tree in sidebar.

Reply:
Sure, will do